_maps/map_files/RandomRuins/SpaceRuins/deepstorage.dmm

```diff
@@ -2666,9 +2666,13 @@
 /area/ruin/space/deepstorage)
 "pk" = (
 /obj/effect/decal/cleanable/dirt,
+
+/obj/item/salvage/ruin/soviet,
```

Comment:
Checks seem to be failing here. Can you double-check what happened here? There seem to be some extra spaces.

Reply:
Handled